### PR TITLE
Intel Gen 4 Xeon and later support bf16

### PR DESCRIPTION
**Pull request by jiqing-feng**
Hi @sayakpaul @patrickvonplaten 

I've found that the example of textual inversion with bf16 mixed-precision only mentioned NV Ampere, but Intel Gen 4 Xeon and later also support bf16, so I added it in the help. Would you please help me to review this change? Thx!

### CODE REVIEW

**Comment by sayakpaul** [comment]
Thanks for your PR! Do you have any results from the Intel board for us to see?

**Comment by jiqing-feng** [comment]
Hi @sayakpaul . The bf16 precision textual inversion on the Intel Sapphire Rapids CPU cluster has been verified, please see the [blog](https://huggingface.co/blog/stable-diffusion-finetuning-intel).

File: examples/textual_inversion/textual_inversion.py [hunk 1]
```diff
@@ -341,7 +341,7 @@ def parse_args():
         help=(
             "Whether to use mixed precision. Choose"
             "between fp16 and bf16 (bfloat16). Bf16 requires PyTorch >= 1.10."
-            "and an Nvidia Ampere GPU."
+            "and Intel Gen 4 Xeon (and later) or Nvidia Ampere GPU."
```
Comment (patrickvonplaten) [inline examples/textual_inversion/textual_inversion.py:344]:
```suggestion
            "and Nvidia Ampere GPU or Intel Gen 4 Xeon (and later) ."
```

Comment (patrickvonplaten) [reply examples/textual_inversion/textual_inversion.py:344]: Let's keep Nvidia Ampere is the first option here

**Comment by jiqing-feng** [comment]
Hi @patrickvonplaten . Thanks for your review. I have fixed it. Would you please help to review it? Thx!

**Comment by jiqing-feng** [comment]
cc @sayakpaul

**Review by sayakpaul — APPROVED** [review]

**Pull request merged**